src/frontend/components/LiveVideoInformationBar/LiveControlButtons/StudentShareDocButton/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useContext } from 'react';
+import { Button, ResponsiveContext } from 'grommet';
+import { defineMessages, useIntl } from 'react-intl';
+
+import { UploadDocSVG } from 'components/SVGIcons/UploadDocSVG';
+
+const messages = defineMessages({
+  ShareDocTitleButton: {
+    defaultMessage: 'Upload file',
+    description: 'Title for the upload file button',
+    id: 'components.StudentShareDocButton.ShareDocTitleButton',
+  },
+});
+
+export const StudentShareDocButton = () => {
+  const size = useContext(ResponsiveContext);
```

Comment:
This is the only button using this `ResponsiveContext`. Should the other use it too or not ?